cypress/support/step-definitions/form-steps.js

```diff
@@ -36,19 +33,19 @@ Then('cancel button is not visible', () => {
 
 Then('buttons are aligned to {string}', (direction) => {
   if (direction === 'right') {
-    buttons(FIRST_ELEMENT).should('have.attr', 'data-component', 'cancel');
-    buttons(SECOND_ELEMENT).should('have.attr', 'data-component', 'form-summary');
-    buttons(SECOND_ELEMENT).should('have.css', 'align-items', 'center')
+    buttons(positionOfElement('second')).should('have.attr', 'data-component', 'cancel');
+    buttons(positionOfElement('third')).should('have.attr', 'data-component', 'form-summary');
+    buttons(positionOfElement('third')).should('have.css', 'align-items', 'center')
```

Comment:
if we use here the same `third` button we could chain the assertion to the previous one